spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'guest show page', type: :feature do
+	let!(:hotel_1) { Hotel.create(name: 'Marriott', location: 'Vail Resort')}
+	let!(:hotel_2) { Hotel.create(name: 'Days Inn', location: 'Portland Airport')}
+	let!(:hotel_3) { Hotel.create(name: 'Holiday Inn', location: 'Los Angeles')}
+
+	let!(:room_5) { hotel_1.rooms.create(rate: 110, suite: "Double") }
+	let!(:room_3) { hotel_1.rooms.create(rate: 140, suite: "Presidential") }
+	let!(:room_2) { hotel_2.rooms.create(rate: 125, suite: "Twin") }
+	let!(:room_4) { hotel_2.rooms.create(rate: 80, suite: "Studio") }
+	let!(:room_1) { hotel_3.rooms.create(rate: 100, suite: "Deluxe") }
+
+	let!(:guest_1) { Guest.create(name: "Charlize Theron", nights: 2) }
+	let!(:guest_2) { Guest.create(name: "John Doe", nights: 1) }
+	let!(:guest_3) { Guest.create(name: "Mary Jane", nights: 5) }
```

Comment:
This data setup is great, you have lots of examples for each type of object. Keep this up.